accounts/abi/bind/template.go

```diff
@@ -154,28 +154,13 @@ var (
 	  TransactOpts bind.TransactOpts    // Transaction auth options to use throughout this session
 	}
 
-	// {{.Type}}Raw is an auto generated low-level Go binding around an Ethereum contract.
-	type {{.Type}}Raw struct {
```

Comment:
You are removing something that doesn't seem to be replaced anywhere else, and it beaks the test. Is that change intentional ?